CONTRIBUTING.md

```diff
@@ -79,20 +77,11 @@ to PyTorch.  For more non-technical guidance about how to contribute to
 PyTorch, see the [Contributing Guide](docs/source/community/contribution_guide.rst).
 
 ## Developing PyTorch
+Follow the instructions for [installing PyTorch from source](https://github.com/pytorch/pytorch#from-source). If you get stuck when developing PyTorch on your machine, check out the [tips and debugging](#tips-and-debugging) section below for common solutions.
 
-A full set of instructions on installing PyTorch from source is here:
-https://github.com/pytorch/pytorch#from-source
-
-To develop PyTorch on your machine, here are some tips:
-
-### Prerequisites
-* CMake. You can install it via `pip install cmake`
-* Python >= 3.7 (3.7.6+ recommended)
```

Comment:
These (and additional) instructions can be found here 
https://github.com/pytorch/pytorch/blob/73e5379fab05c40ff6f42500309cdc17ee57548c/README.md?plain=1#L182-L188